### PR TITLE
feat: Added support for activities with multilingual chars

### PR DESCRIPTION
**Pull request by tomaskatz96**

### CODE REVIEW

**Comment by mykola-mokhnach** [comment]
Thank you @tomaskatz96 

Please sign the CLA and add a unit test to verify your change

**Comment by tomaskatz96** [comment]
Hi @mykola-mokhnach,
Added tests and signed the CLA

File: test/unit/helper-specs.js [hunk 1]
```diff
@@ -472,4 +472,16 @@ describe('helpers', withMocks({fs}, function (mocks) {
...
   });
+  describe('matchComponentName', function () {
+    it('test valid activity name', function () {
+      const activity = "ןذأצЮυπиС.נפשוקשΤπΟ.ЦοКسئοهΦΦ";
+      const names = matchComponentName(activity);
+      names.should.eql(["ןذأצЮυπиС.נפשוקשΤπΟ.ЦοКسئοهΦΦ"]);
+    });
+    it('test invalid activity name', function () {
+      const activity = "User@123";
+      const result = String(matchComponentName(activity));
```
Comment (mykola-mokhnach) [inline test/unit/helper-specs.js:483]: _.isNull(matchComponentName(activity)).should.be.true;

**Comment by mykola-mokhnach** [comment]
> Added tests and signed the CLA

Could you please maybe squash commits and/or make sure they were done using the same email address that was used to sign the CLA?

File: test/unit/helper-specs.js [hunk 2]
```diff
@@ -472,4 +472,16 @@ describe('helpers', withMocks({fs}, function (mocks) {
       names.should.be.eql([]);
     });
   });
+  describe('matchComponentName', function () {
+    it('test valid activity name', function () {
+      const activity = "ןذأצЮυπиС.נפשוקשΤπΟ.ЦοКسئοهΦΦ";
```
Comment (mykola-mokhnach) [inline test/unit/helper-specs.js:477]: Please use single quotes everywhere

File: test/unit/helper-specs.js (hunk 1 above)
Comment (tomaskatz96) [reply test/unit/helper-specs.js]: Correct, thank you.
Done

**Comment by tomaskatz96** [comment]
@mykola-mokhnach Do I need to open new PR with all the changes and close this one?

**Comment by KazuCocoa** [comment]
Probably it makes sense to create a new one, although, the CLA checks the commit user's email. Please make sure your commit email is the same as what you use for the sign.

e.g.

> $ git log
> commit 4f1e8d0f76f128ba1567cf605d8128f9466a163f (grafted, HEAD -> add_multilingual_support, origin/add_multilingual_support)
> Author: tomaskatz96 <67865483+tomaskatz96@users.noreply.github.com>

**Pull request closed without merging**